### ISSUE TITLE
bamCaverage output .bw file can't  be viewed correct by IGV

### ISSUE THREAD

**Issue by ShaokunPan** [deeptools/deepTools#1150]
Hi

I use the sorted .bam file as input and .bw file is the output, there was no error reported, when using bamCaverage.
The sorted .bam file can be viewed using IGV corrected, but the .bw file can't be viewed and no error reported.
Can someone help me? Thanks!

![微信截图_20220721094944](https://user-images.githubusercontent.com/45087357/180112222-fa3da4e1-dfa7-44ba-bc3d-627e6030d62f.png)

deeptools 3.5.1
python: 3.9.3
IGV: 2.12.0

bamCoverage --bam ./sort/OM8_S8_2019nCoV.mapped.sort.bam \
	 -o OM8_S8_2019nCoV.SeqDepth.bw \
	 --outFileFormat bigwig \
	 -p 20 \
	 --binSize 1

log:
bamFilesList: ['./sort/OM8_S8_2019nCoV.mapped.sort.bam']
binLength: 1
numberOfSamples: None
blackListFileName: None
skipZeroOverZero: False
bed_and_bin: False
genomeChunkSize: None
defaultFragmentLength: read length
numberOfProcessors: 20
verbose: False
region: None
bedFile: None
minMappingQuality: None
ignoreDuplicates: False
chrsToSkip: []
stepSize: 1
center_read: False
samFlag_include: None
samFlag_exclude: None
minFragmentLength: 0
maxFragmentLength: 0
zerosToNans: False
smoothLength: None
save_data: False
out_file_for_raw_data: None
maxPairedFragmentLength: 1000

**Comment by dpryan79** [comment]
Can you post the bigWig somewhere so I can more easily debug this?

**Comment by aslisadli** [comment]
I am experiencing the same issue. What was the solution for that problem? Thanks.

**Comment by dpryan79** [comment]
@aslisadli There was never any follow-up from the original poster to debug the cause. Can you make your bigWig available to us to debug this?

**Comment by dpryan79** [comment]
Please upload the file to dropbox or google drive or similar and send a link to dpryan79@gmail.com

**Comment by dpryan79** [comment]
I downloaded one of the yeast chromosomes from NCBI and the most recent version of IGV. I can't seem to reproduce the error:

![image](https://user-images.githubusercontent.com/6399000/231280979-94d1353b-4ed5-45af-a23c-115b6e2734d4.png)

**Comment by aslisadli** [comment]
Thanks a lot. I tried the single chromosome fasta download using the INSDC
code GenBank: BK006935.2 and it worked. However when I tried to use the
genome already in IGV system it does not work.
Finally I figured out that I needed to use indexing my reference using the
".fasta" file not the ".fna". Fna files gives chromosomes different names
that does not match with IGV hosted genomes.

Thanks again for your help
